shanoir-ng-datasets/src/main/java/org/shanoir/ng/importer/service/ImporterService.java

```diff
@@ -121,12 +140,17 @@
 	@Autowired
 	RabbitTemplate rabbitTemplate;
 
+	@Autowired
+	SolrService solrService;
+
 	private static final String SESSION_PREFIX = "ses-";
 
 	private static final String SUBJECT_PREFIX = "sub-";
 
 	private static final String EEG_PREFIX = "eeg";
 	
+	private static final String PROCESSED_DATASET_PREFIX = "processed-dataset";
+
 	private static int instancesCreated = 0;
```

Comment:
Hi Jean-Come, I think, this could be removed please. Thank you, Michael

Reply:
PROCESSED_DATASET_PREFIX is used later for the nifti folder.

instancesCreated is also used in createAllDatasetAcquisition even if I don't really understand its purpose.
The commit states: Fix multi-threading bug with datasetAcquisitionStrategy + Merge with develop

Maybe it was a debug log so I can remove all of it ?

Reply:
I guess it was a debug log maybe, hope it was not me who introduced it...